cores/esp8266/flash_hal.cpp

```diff
@@ -60,7 +60,7 @@ int32_t flash_hal_erase(uint32_t addr, uint32_t size) {
     const uint32_t sector = addr / SPI_FLASH_SEC_SIZE;
     const uint32_t sectorCount = size / SPI_FLASH_SEC_SIZE;
     for (uint32_t i = 0; i < sectorCount; ++i) {
-        optimistic_yield(10000);
+        optimistic_yield(1000);
```

Comment:
Here things are different because erasing is sloooow (many x the write or read times) and this call can do multiple sectors.  No issue decreasing yield time here, I suppose.

Reply:
But no compelling reason to do it, either - just slows down things even more, doesn't it? I'll revert to the previous 10000µs.